pkg/virt-api/webhooks/validating-webhook/admitters/vmirs-admitter_test.go

```diff
@@ -88,9 +110,14 @@ var _ = Describe("Validating VMIRS Admitter", func() {
 				Selector: &metav1.LabelSelector{
 					MatchLabels: map[string]string{"match": "this"},
 				},
-				Template: newVirtualMachineBuilder().WithDisk(v1.Disk{
-					Name: "testdisk",
-				}).BuildTemplate(),
+				Template: buildVMITemplate(
+					libvmi.New(
+						libvmi.WithName("testvmi"),
+						withDisk(v1.Disk{
+							Name: "testdisk",
+						}),
```

Comment:
For the purpose of this test you could simplify this to be `withTestDisk`. It is used only in this place.